crates/astria-sequencer/Cargo.toml

```diff
@@ -38,6 +38,7 @@ tower-http = { version = "0.4", features = ["cors"] }
 async-trait = { workspace = true }
 bytes = { workspace = true }
 divan = { workspace = true, optional = true }
+ethers = { workspace = true, features = ["ws"] }
```

Comment:
We don't want to have this as a dependency. It's fine if the hex transaction hashes don't have "0x" in front of them.

Reply:
Removed and reverted back to `hex::encode` in 1fd6ad2.